### PR TITLE
create a service to get query params

### PR DESCRIPTION
**Pull request by itkg-bfouche**

### CODE REVIEW

**Comment by nicolasThal** [comment]
Is the unit test missing?

**Comment by itkg-bfouche** [comment]
@nicolasThal

File: FrontBundle/Tests/Manager/ParametersManagerTest.php [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+        return array(
+            array('fr', 'demoSite', 'android', array()),
+            array('fr', 'demoSite', null, array('_controller' => 'OpenOrchestra\FrontBundle\Controller\NodeController::showAction', '_locale' => 'fr', '_route' => '2_552e2d3802b')),
+            array('fr', 'demoSite', '', array('_controller' => 'OpenOrchestra\FrontBundle\Controller\NodeController::showAction', '_locale' => 'fr', '_route' => '2_552e2d3802b', '_route_params' => array('_locale' => 'fr', 'nodeId' => 'root', 'siteId' => 'demoSite'))),
+            array('en', 'site', 'ios', array('_controller' => 'OpenOrchestra\FrontBundle\Controller\NodeController::showAction', '_locale' => 'fr', 'siteId' => 'site2','_route' => '2_552e2d3802b', '_route_params' => array('_locale' => 'fr', 'nodeId' => 'root', 'siteId' => 'demoSite'))),
+        );
+    }
+
+}
+
```
Comment (nicolasThal) [inline FrontBundle/Tests/Manager/ParametersManagerTest.php]: :hamburger:

File: FrontBundle/Manager/ParametersManager.php [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+ */
+class ParametersManager
+{
+    /**
+     * @param Request           $request
+     * @param ReadNodeInterface $node
+     *
+     * @return array
+     */
+    public function getParameters(Request $request, ReadNodeInterface $node)
```
Comment (nicolasThal) [inline FrontBundle/Manager/ParametersManager.php]: generate for the method name.

The class name already indicates what it does

File: FrontBundle/Manager/ParametersManager.php [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+namespace OpenOrchestra\FrontBundle\Manager;
+
+use OpenOrchestra\ModelInterface\Model\ReadNodeInterface;
+use Symfony\Component\HttpFoundation\Request;
+
+/**
+ * Class ParametersManager
+ */
+class ParametersManager
```
Comment (nicolasThal) [inline FrontBundle/Manager/ParametersManager.php]: you can also rename the class SubQueryParameterManager

**Comment by itkg-bfouche** [comment]
@nicolasThal

**Comment by nicolasThal** [comment]
:+1:

**Pull request merged**